src/nvme/ioctl.c

```diff
@@ -858,6 +858,13 @@ int nvme_get_log_zns_changed_zones(int fd, __u32 nsid, bool rae,
 			    NULL);
 }
 
+int nvme_media_unit_stat_log(int fd,  __u16 domid, struct nvme_media_unit_stat_log *mus)
+{
+	return nvme_get_log(fd, NVME_LOG_LID_MEDIA_UNIT_STATUS, NVME_NSID_NONE, 0,
+		NVME_LOG_LSP_NONE, domid, 0, NVME_UUID_NONE, NVME_CSI_NVM, false, 
```

Comment:
- whitespace at the end of line
- align the start of the line to the (
-

Reply:
Addressed
Thanks @igaw